resources/parser.rb

```diff
@@ -2,7 +2,7 @@
 # Cookbook:: syslog_ng
 # Resource:: parser
 #
-# Copyright:: 2019, Ben Hughes <bmhughes@bmhughes.co.uk>
+# Copyright:: 2020, Ben Hughes <bmhughes@bmhughes.co.uk>
```

Comment:
Usually, you make this a range from the earliest, but honestly you can just leave the date off entirely and then you never need to update it. Copyright is longer then your lifetime.

Reply:
Good to know, I'll remove it then I don't have to worry about it. Thanks!